core/ustring.cpp

```diff
@@ -120,9 +120,13 @@ void String::copy_from(const char *p_cstr) {
 void String::copy_from(const CharType *p_cstr, int p_clip_to) {
 
 	int len = 0;
-	const CharType *ptr = p_cstr;
-	while (*(ptr++) != 0)
-		len++;
+
+	if (p_cstr) {
```

Comment:
Look at the code below:
```c++
int len = 0;

if (p_cstr) {

    const CharType *ptr = p_cstr;
    while (*(ptr++) != 0)
        len++;
}

if (p_clip_to >= 0 && len > p_clip_to)
    len = p_clip_to;

if (len == 0) {

    resize(0);
    return;
}
```
The variable `len` will keep `0` if the value of `p_cstr` is `NULL`. So we don't need to do that :)